oauth2-authorization-server/src/main/java/org/springframework/security/oauth2/server/authorization/token/OAuth2TokenResult.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.oauth2.server.authorization.token;
+
+import org.springframework.security.oauth2.core.AbstractOAuth2Token;
+
+/**
+ * Result returned by a {@link OAuth2TokenIssuer}, a combination of a token of {@code TOKEN_TYPE} type and {@link OAuth2TokenMetadata}.
+ *
+ * @see OAuth2TokenIssuer
+ * @see OAuth2TokenMetadata
+ *
+ * @author Alexey Nesterov
+ * @since 0.0.4
```

Comment:
Change to `0.1.0`. Also, please ensure it's added in all other new classes as I noticed it was missing in a couple.